classes/Mail.php

```diff
@@ -49,22 +49,53 @@ class MailCore extends ObjectModel
     /**
      * @see ObjectModel::$definition
      */
-    public static $definition = array(
+    public static $definition = [
         'table' => 'mail',
         'primary' => 'id_mail',
-        'fields' => array(
-            'recipient' => array('type' => self::TYPE_STRING, 'validate' => 'isEmail', 'copy_post' => false, 'required' => true, 'size' => 126),
-            'template' => array('type' => self::TYPE_STRING, 'validate' => 'isTplName', 'copy_post' => false, 'required' => true, 'size' => 62),
-            'subject' => array('type' => self::TYPE_STRING, 'validate' => 'isMailSubject', 'copy_post' => false, 'required' => true, 'size' => 254),
-            'id_lang' => array('type' => self::TYPE_INT, 'validate' => 'isUnsignedId', 'copy_post' => false, 'required' => true),
-            'date_add' => array('type' => self::TYPE_DATE, 'validate' => 'isDate', 'copy_post' => false, 'required' => true),
-        ),
-    );
+        'fields' => [
+            'recipient' => [
+                'type' => self::TYPE_STRING,
+                'validate' => 'isEmail',
+                'copy_post' => false,
+                'required' => true,
+                'size' => 126
+            ],
+            'template' => [
+                'type' => self::TYPE_STRING,
+                'validate' => 'isTplName',
+                'copy_post' => false,
+                'required' => true,
+                'size' => 62
+            ],
+            'subject' => [
+                'type' => self::TYPE_STRING,
+                'validate' => 'isMailSubject',
+                'copy_post' => false,
+                'required' => true,
+                'size' => 254
+            ],
+            'id_lang' => [
+                'type' => self::TYPE_INT,
+                'validate' => 'isUnsignedId',
+                'copy_post' => false,
+                'required' => true
+            ],
+            'date_add' => [
+                'type' => self::TYPE_DATE,
+                'validate' => 'isDate',
+                'copy_post' => false,
+                'required' => true
+            ],
+        ],
+    ];
 
     const TYPE_HTML = 1;
     const TYPE_TEXT = 2;
     const TYPE_BOTH = 3;
 
+    const METHOD_SMTP = 2;
+    const METHOD_DISABLE = 3;
```

Comment:
please explain this const, I was required to read code to understand it

Reply:
Done!